bears/java/CheckstyleBear.py

```diff
@@ -25,6 +27,20 @@ class CheckstyleBear:
 
     LANGUAGES = "Java"
 
-    @staticmethod
-    def create_arguments(filename, file, config_file):
-        return '-jar', checkstyle_jar_file, '-c', google_checks, filename
+    known_configs = {
+        "google": os.path.join(DIR_PATH, 'google_checks.xml')}
+
+    def create_arguments(self, filename, file, config_file,
+                         checkstyle_configs: path="google"):
```

Comment:
not so sure, if I provide the value `google`, the path function will make an absolute path out of it